docker-compose.yml

```diff
@@ -146,9 +146,9 @@ services:
       - IMPORT_SCHEDULE=${IMPORT_SCHEDULE}
       - IMPORTER_CRON_ENABLE=${IMPORTER_CRON_ENABLE:-false}
       - IMPORTER_AGENT=${IMPORTER_AGENT}
-      - MEDIAWIKI_API_URL=${WIKIBASE_SCHEME:-https}://mardi-wikibase/w/api.php
+      - MEDIAWIKI_API_URL=http://mardi-wikibase/w/api.php
```

Comment:
This is effectively the same as setting WIKIBASE_SCHEME to HTTP. Why are you not doing that?

Reply:
Yes, but the SCHEME is set to https in production, which results in the route https://mardi-wikibase failing because internally, mardi-wikibase is only at port 80. The other option would be to use https://portal.mardi4nfdi.de as the url for these API calls, but I cannot access either port 443 from within the container, so that fails (same problem we have in the wdqs).